src/service.py

```diff
@@ -29,18 +31,30 @@
 }
 
 
+# TODO: more medium term it would feel less weird to initialise this
+# object with a 'settings' object so that the annex bit can be done
+# more tidily.  But this is pulled into the other modules as "from
+# service import service" so that's not going to directly work!
+#
+# However, this is actually really hard to do correctly - even passing
+# 'settings' through as an argument to status() is hard because it's
+# used to _load_ the settings and confirm that all settings are filled
+# out appropriately!
 class MontaguService:
     def __init__(self):
         self.client = docker.from_env()
 
-    @property
```

Comment:
revert back to being a property and drop the settings check here - we'll move from a global status to one parameterised by the settings in a separate PR